grpc/interceptors_test.go

```diff
@@ -161,3 +170,54 @@ func TestTimeouts(t *testing.T) {
 		})
 	}
 }
+
+func TestRequestTimeTagging(t *testing.T) {
+	clk := clock.NewFake()
+	// Listen for TCP requests on a random system assigned port number
+	lis, err := net.Listen("tcp", ":0")
+	if err != nil {
+		log.Fatalf("failed to listen: %v", err)
+	}
+	// Retrieve the concrete port numberthe system assigned our listener
+	port := lis.Addr().(*net.TCPAddr).Port
+
+	// Create a new ChillerServer
+	serverMetrics := NewServerMetrics(metrics.NewNoopScope())
+	si := newServerInterceptor(serverMetrics, clk)
+	s := grpc.NewServer(grpc.UnaryInterceptor(si.intercept))
+	test_proto.RegisterChillerServer(s, &testServer{})
+	// Chill until ill
+	go func() {
+		start := time.Now()
+		if err := s.Serve(lis); err != nil &&
+			!strings.HasSuffix(err.Error(), "use of closed network connection") {
```

Comment:
We can maybe use `cmd.FilterShutdownErrors` to simplify this (if looping dependencies doesn't catch us; I think it might not, for tests?)

Reply:
I cribbed this from `TestTimeouts`, perhaps unnecessarily?